deel-2-flexbox-layout/opdracht-2/styles/styles.css

```diff
@@ -1 +1,32 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+/*Schrijf hier jouw CSS*/
+
```

Comment:
je bent in deze opdracht de grow en shrink voorwaarden vergeten per item. 
omdat ze een automatische waarde hebben (0 voor grow en 1 voor shrink) dien je ze mee te geven als ze hiervan afwijken (de voorwaarden per opdracht geven aan wat ze wel/niet mogen)